### PR TITLE
fix: switching to ww cause scrolling (close #194)

### LINKED ISSUE #194: Switching to wysiwyg mode move to the top of the page
## Version
1.1.1

## Test Environment
Windows 7 and Chrome 66

## Current Behavior
Switching to wysiwyg mode induce a move to the top of the page if you can scroll vertically.

## Expected Behavior
Switching to wysiwyg mode should'nt change view.

### PR DESCRIPTION
**Pull request by kyuwoo-choi**
### Please check if the PR fulfills these requirements
- [x] It's the right issue type on the title
- [x] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [x] The commit message follows our guidelines
- [x] Tests for the changes have been added (for bug fixes/features)
- [x] Docs have been added/updated (for bug fixes/features)
- [x] It does not introduce a breaking change or has a description of the breaking change

### Description
#194

---
Thank you for your contribution to TOAST UI product. 🎉 😘 ✨

### CODE REVIEW

File: src/js/wysiwygEditor.js [hunk 1]
```diff
@@ -1009,16 +991,33 @@ class WysiwygEditor {
...
+   * move scroll to cursor
+   * scrollIntoView browser function may cause scrolling on document.
+   * this function aims to replace scrollIntoView function to prevent that.
+   * it will move the scroll of squire only.
+   * @memberof SquireExt
+   */
+  scrollIntoCursor() {
+    const scrollTop = this.scrollTop();
+    const {top: cursorTop, height: cursorHeight} = this.getEditor().getCursorPosition();
+    const {top: editorTop, height: editorHeight} = this.$editorContainerEl.get(0).getBoundingClientRect();
```
Comment (codepink) [inline src/js/wysiwygEditor.js]: 사소한거지만.. 개행하면 좀 더 가독성에 좋지 않을까요?ㅎ
es6는 컨벤션이 타이트하지 않아서.. 개취입니다만ㅎㅎ
```js
const {
  top: editorTop,
  height: editorHeight
} = this.$editorContainerEl.get(0).getBoundingClientRect();
```

**Comment by codepink** [comment]
[7/20] 리뷰 완료하였습니다. 고생하셨어요~

File: src/js/wysiwygEditor.js (hunk 1 above)
Comment (kyuwoo-choi) [reply src/js/wysiwygEditor.js]: :) 동의합니다.

**Comment by jinwoo-kim-nhn** [comment]
이견 없습니다. 고생하셨습니다.

File: src/js/wysiwygEditor.js [hunk 2]
```diff
@@ -490,7 +490,7 @@ class WysiwygEditor {
       }
 
       this.defer(() => {
-        this._scrollToRangeIfNeed();
+        this.scrollIntoCursor();
```
Comment (shiren) [inline src/js/wysiwygEditor.js]: 중괄호 빼고 한줄로 해도 될것같아요

Comment (kyuwoo-choi) [reply src/js/wysiwygEditor.js]: 옙!

**Pull request merged**